### PR TITLE
Add sentry competitor doc

### PR DESCRIPTION
**Pull request by jay-khatri**
## Summary

Adds a sentry competitor doc.

## How did you test this change?

clicktest

## Are there any deployment considerations?

n/a

### CODE REVIEW

**Review by Vadman97 — APPROVED** [review]

File: highlight.io/components/Competitors/sentry.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	HighlightIsOpenAndTransparent,
+	HighlightShipsNewFeatures,
+} from './shared-snippets'
+
+export const SentrySpec: Competitor = {
+	name: 'Sentry',
+	header: 'A Truly Open Source Sentry Alternative.',
+	subheader:
+		'Session replay of your frontend, fullstack error monitoring, and powerful logging.',
+	logoDesktop: datadogfull,
```
Comment (et) [inline highlight.io/components/Competitors/sentry.ts:14]: Looks like we need to s/datadog/sentry/g in this file.

**Review by et — CHANGES_REQUESTED** [review]

File: highlight.io/components/Competitors/sentry.ts (hunk 1 above)
Comment (jay-khatri) [reply highlight.io/components/Competitors/sentry.ts]: thanks. added the logos.

File: highlight.io/components/Competitors/sentry.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	paragraphs: [
+		HighlightIsOpenAndTransparent('Sentry'),
+		HighlightShipsNewFeatures,
+		{
+			header: 'Full-stack Observability',
+			body: 'While Sentry provides a comprehensive set of separate features for error monitoring and session replay, it does not support logging. [Highlight.io](https://highlight.io) enables teams to monitor and optimize their entire tech stack, pairing server-side infrastructure with your frontend web applications. This makes Highlight.io a more comprehensive solution for developers who need to monitor their entire tech stack, in a simple, easy to implement solution.',
+		},
+		{
+			header: 'Agent-less Architecture',
+			body: `Datadog's architecture requires you to install an agent on your server. This can be a pain to install and maintain. Highlight.io's architecture is agent-less and requires no installation, but in the case that you want to install an agent long-term, we have an open-source agent/collector that you can install on your servers.`,
```
Comment (et) [inline highlight.io/components/Competitors/sentry.ts:154]: Remove this as well.

File: highlight.io/components/Competitors/sentry.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+			title: 'General',
+			rows: [
+				{
+					feature: 'Unlimited Team Members',
+					highlight: 1,
+					competitor: 1,
+				},
+				{
+					feature: 'Support Ticket Integrations',
+					tooltip: 'Create tickets with clickup, linear, or jira.',
```
Comment (et) [inline highlight.io/components/Competitors/sentry.ts:27]: We don't support jira (yet).

`Create tickets with Linear, GitHub, Height, or ClickUp`

File: highlight.io/components/Competitors/sentry.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+					feature: 'Canvas & WebGL Recording',
+					tooltip:
+						'Ability to record canvas and webgl elements in your application.',
+					highlight: 1,
+					competitor: 0,
+				},
+				{
+					feature: 'Shadow DOM Recording',
+					tooltip:
+						'Ability to record shadow dom elements in your application.',
```
Comment (et) [inline highlight.io/components/Competitors/sentry.ts:88]: `shadow DOM`

File: highlight.io/components/Competitors/sentry.ts [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+				},
+				{
+					feature: 'Embedded, fullstack error monitoring',
+					highlight: 1,
+					competitor: 1,
+				},
+				{
+					feature: 'Canvas & WebGL Recording',
+					tooltip:
+						'Ability to record canvas and webgl elements in your application.',
```
Comment (et) [inline highlight.io/components/Competitors/sentry.ts:81]: `and WebGL elements`

**Pull request merged**